src/dlgAboutDialog.cpp

```diff
@@ -877,6 +901,30 @@ dlgAboutDialog::dlgAboutDialog(QWidget* parent) : QDialog(parent)
     license_3rdParty_texts.append(QStringLiteral("</body></html>"));
 
     textBrowser_license_3rdparty->setHtml(license_3rdParty_texts.join(QString()));
-
     // clang-format on
 }
+
+void dlgAboutDialog::setSupportersTab(const QString& htmlHead) const
+{
+    QString supporters_text(QStringLiteral(R"(
+               <p><div style="text-align: center">%1</a>
```

Comment:
Anchor end tag is obsolete

Reply:
OK fixed, but the whole thing does not look correct anyway.

Reply:
> Anchor end tag is obsolete

Are you sure - remember this is Qt's own *Rich-text* it is **NOT** HTML5...!

Reply:
Sure. Look at content of %1